.circleci/config.yml

```
@@ -87,6 +87,7 @@ job-references:
 
   php_core_job: &php_core_job
     steps:
+      - run: sudo apt-get update && sudo apt-get -y install git ssh
```

Maybe we can make this cleaner in the CircleCI job output:

```suggestion
      - run:
          name: Add missing packages
          command: sudo apt-get update && sudo apt-get -y install git ssh
```